tutor/src/screens/scores-report/assignment-header.jsx

```diff
@@ -26,15 +27,15 @@ const ReviewLink = (props) => {
 }
 
 
-const AverageLabel = (props) => {
-  if (props.average_score) {
+const AverageLabel = ({ heading }) => {
+  if (heading.isDue && !isNil(heading.average_score)) {
     return (
       <span className="average">
-        {(props.average_score * 100).toFixed(0)} %
+        {(heading.average_score * 100).toFixed(0)}%
       </span>
     );
-  } if (props.heading.type === 'external') {
-    const p = props.heading.average_progress;
+  } if (heading.type === 'external') {
+    const p = heading.average_progress;
     let percent;
     if (p < 1 && p > 0.99) {
       percent = 99; // Don't round to 100% when it's not 100%!
```

Comment:
this was never working as far as I can tell, since `average_score` was on the `heading` not on props